### PR TITLE
Implemented Pagination In Condition Widget.

### PR DESCRIPTION
**Pull request by ujjwaltyagi355**
The Pull Request is for issue [MF-469](https://issues.openmrs.org/browse/MF-469).
I have implemented Pagination in the Condition Widget as per the design requirements for the issue.
Do have a look and tell me if any changes need to be made. 
Thank you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "@fortawesome/react-fontawesome": "^0.1.11",
     "carbon-components-react": "^7.25.0",
     "d3": "^5.16.0",
+    "import-map-overrides": "^2.3.1",
```
Comment (FlorianRappl) [inline package.json]: Why did you add this dependency?
Should be removed

File: src/widgets/conditions/conditions-overview.component.tsx [hunk 2]
```diff
@@ -132,34 +132,28 @@ const ConditionsOverview: React.FC<ConditionsOverviewProps> = () => {
...
-                      )}
                   </TableBody>
                 </Table>
               )}
             </DataTable>
           </TableContainer>
+          {totalRows > conditionsToShowCount && (
+            <Pagination
+              totalItems={conditions.length}
+              backwardText="Previous page"
```
Comment (FlorianRappl) [inline src/widgets/conditions/conditions-overview.component.tsx]: Translations for these strings would be superb.

**Review by FlorianRappl — COMMENTED** [review]
I think we are on the right track here.

File: src/widgets/conditions/conditions-overview.component.tsx (hunk 2 above)
Comment (ujjwaltyagi355) [reply src/widgets/conditions/conditions-overview.component.tsx]: Sure, I will make the changes @FlorianRappl .

File: package.json (hunk 1 above)
Comment (ujjwaltyagi355) [reply package.json]: Sure. Will remove them.

**Comment by ujjwaltyagi355** [comment]
Hello @FlorianRappl,
Do have a look and tell me if any changes need to be made.
Thank you.
I just had one doubt, should I make a separate PR for each widget where pagination will be implemented, or I just commit changes to this one only?

**Review by FlorianRappl — APPROVED** [review]
LGTM! :beers:
